packages/select-rich/src/LionSelectInvoker.js

```diff
@@ -13,7 +13,7 @@ import { css, html, SlotMixin } from '@lion/core';
 export class LionSelectInvoker extends SlotMixin(LionButton) {
   static get styles() {
     return [
-      ...super.styles,
+      ...LionButton.styles,
```

Comment:
can't we use super here?

Reply:
Let me check

Reply:
Thanks, Would be hard though when the parent is a Mixin. 
Does the transpilation issue also occur in non Typescript code (https://github.com/lit/lit.dev/pull/535#discussion_r719905974 mentions TS explicitly)?

Reply:
Ok, apparently it triggers TS linting errors (which we get via jsdoc annotations), so should be good as a workaround 👍

Reply:
I added a TODO to change it back once fixed.